### PR TITLE
Uncertain fragments check optimization

### PR DESCRIPTION
**Pull request by khoidt**

### CODE REVIEW

**Comment by khoidt** [comment]
[Create new endpoint for uncertain fragments in Chapters](https://trello.com/c/Hg3Lx7BV/598-create-new-endpoint-for-uncertain-fragments-in-chapters)

File: src/corpus/ui/Chapters.tsx [hunk 1]
```diff
@@ -44,12 +44,19 @@ const Manuscripts = withData<
   {
     uncertainFragments: readonly UncertainFragment[]
     textService
```
Comment (fsimonjetz) [inline src/corpus/ui/Chapters.tsx:46]: Maybe add types `TextService` and `FragmentService`?

File: src/corpus/ui/Chapters.tsx [hunk 2]
```diff
@@ -169,7 +176,17 @@ const Manuscripts = withData<
                   <ul className="list-of-manuscripts__uncertain-fragments">
                     {uncertainFragments.map((fragment, index) => (
                       <li key={index}>
-                        <FragmentariumLink item={fragment} />
+                        <FragmentariumLink
+                          item={{
```
Comment (fsimonjetz) [inline src/corpus/ui/Chapters.tsx:180]: Would it be more readable if we extract it to `const item = ...` and then call `<FragmentariumLink item={item} />`?

Comment (yCobanoglu) [reply src/corpus/ui/Chapters.tsx:180]: `uncertainFragment` instead `fragment`

**Comment by yCobanoglu** [comment]
As discussed we get rid of uncertainFragments property and make the new behaviour as the default (not checking for isInFragmentarium only checking in frontend when necessary).

**Pull request merged**